t/05-scheduler-full.t

```diff
@@ -210,7 +210,7 @@ subtest 're-scheduling and incompletion of jobs when worker rejects jobs or goes
     wait_for_worker($schema, 5);
 
     note 'waiting for job to be incompleted';
-    for (0 .. 100) {
+    for (0 .. 200) {
```

Comment:
Please get rid of the magic number make this fail clearly when the condition isn't met.
```suggestion
    wait_for_or_bail_out { $jobs->find(99982)->state eq OpenQA::Jobs::Constants::DONE } 'Job 99982 is done';
```

Reply:
What do you suggest?

Reply:
This always results in a failed test.
The `wait_for_or_bail_out` doesn't really seem to do anything.

Reply:
I think it is a better idea to leave this in but be more verbose to see if the job is behaving the same as locally in a case that actually results in a sporadic error.

Reply:
What do you mean by "doesn't do anything"? The function waits until the condition is fulfilled after repeatedly calling it. You can override `timeout` and `interval`, which is 60/.1 by default

Reply:
It just returns immediately which of course breaks the three checks afterwards.

Reply:
But oddly I can't seem to reproduce that now anymore.